app/bundles/CoreBundle/Controller/AbstractFormController.php

```diff
@@ -166,7 +166,9 @@ protected function isFormValid(Form $form, array $data = null)
         //bind request to the form
         $form->handleRequest($this->request);
 
-        return $form->isValid();
+        if ($form->isSubmitted()) {
+            return $form->isValid();
+        }
```

Comment:
The method should return bool. With the suggested change there is a chance the method won't return anything. I'd suggest this:
```suggestion
        return $form->isSubmitted() && $form->isValid();
```

Reply:
@escopecz I agree with you, In the first comment I made the function return null, then I removed it. I think your solution is better to make sure we always return the right value.

Thanks